### ISSUE TITLE
[SW-Content] Intake- FSR's 'Find a form' updated content review from the Debt Resolution Team

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#24443]
### Content support for:
- Unauthenticated (static) VA.gov pages
- Apps or tools
- Other writing or editing requests

**Note: These requests are handled by the Sitewide Content Team.** 

## How to submit your request

- [ ] Complete each area of this intake form that applies to your team's needs. Provide as much detail as you can so we can fully understand your request.
- [ ] Determine if this request is for:
     - [x] Content Only      
     - [ ] Public Website Engineering Support      
     - [ ] Both (Content + PW Eng Spt)   
- [ ] Post a link to this issue in the appropriate channel: For Engineering only requests: [#vsa-public-websites](https://dsva.slack.com/channels/vsa-public-websites), and in slack tag @brianalloyd, @ncksllvn and @kelsonic. Content only support: [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) in Slack, tag @RLHecht.  In situations where both are needed post to Public Websites and in Slack tag @brianalloyd, @ncksllvn, @kelsonic and @RLHecht for awareness.  
- [ ] If noted in the guidance below, please schedule any relevant meetings.

#### Content requests

|Type of need | When to submit your request |
| :-- | :-- |
| **New unauthenticated VA.gov page (such as a static landing page to support a new app/tool)** | At project kickoff |
| **Updates to an existing unauthenticated VA.gov page** | For significant updates (like consolidating pages or reworking page content): At project kickoff <br> For smaller updates (like adding a section of new info): At least 1 full sprint in advance (so we can work it into our next sprint). |
| **Writing support for app/tool** | At project kickoff | 
| **Editing support for app/tool or other writing/editing support** | For multiple screens: At least 1 full sprint in advance (so we can work it into our next sprint). <br> For very small needs (like an error message): At least 2 days in advance | 

### Engineering requests

Please submit all engineering requests **at least 1 full sprint in advance** (so we can work it into our next sprint).

---

## About your app/tool/functionality

* **Is this request related to a VA.gov tool? If yes, provide a brief description of the tool or functionality:**  
   * The Financial Status Report is launching soon, so we'd like to update the content on [the FSR's 'Find a form' page](https://www.staging.va.gov/find-forms/about-form-5655/). 
* **Provide links to any available collateral: product briefs/outlines, wireframes, prototypes, user research synthesis:** 
   * [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/Medical_Copays/Product_Outline_Medical_Copays.md)
   * [Research folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/Debt%20Resolution/Medical_Copays/research)
   * [FSR in staging](https://staging.va.gov/manage-va-debt/request-debt-help-5655/introduction)

---

## About your support needs
*Please select all that apply, and follow related guidance.* 

### Content
     
- [ ] **New unauthenticated VA.gov page**
   * **Additional info.** Please provide any details not covered above about what app/tool/other need this new page will support. 
      * < URL list >
   * **Schedule a meeting.** Please schedule a meeting with @RLHecht and @brianalloyd to kick off this work. We'll work with your team and the VSP IA resource to determine how best to accommodate this new page.  

- [x] **Updates to an existing unauthenticated VA.gov page**
   * **What URL(s) do we need to review or update?** 
      * https://www.va.gov/find-forms/about-form-5655/
   * **Are updates for a small section of copy?** If yes, please tell us what new info we need to consider and what (if anything) it's replacing.
We'd like to add sections to the FSR's results page in alignment with [VA form 526-EZ's result page](https://www.staging.va.gov/find-forms/about-form-21-526ez/). I've attached a Word doc with proposed content below.
   * **Are updates related to usability testing or analytics?** If yes, please provide details.**
       * < list of new info >

- [ ] **Writing support for new app/tool** 
   * **Additional info.** Please provide any details not covered above about what app/tool/other need this new page will support. 
      * < URL list >
   * **Schedule a meeting.** Please schedule a meeting with @RLHecht and @brianalloyd to kick off this work. We'll work with your team and the VSP content resource to determine how best to support your needs.   

- [x] **Editing support for app/tool** <br> <br> **Note:** We'll work with your team and the VSP content resource to determine how best to support your needs.
   * URL where draft content appears (if available) <br>    
     - < link > 
   * **Word document with draft content** (if needed) <br> **Note:** If you have more than 2 paragraphs of text to edit, please provide text in a Word doc.     
     - [Word doc with content](https://app.zenhub.com/files/133843125/e27d1880-159b-4b6f-b3f8-78f187917fe5/download)    
   * **Design context** <br> **Example:** Is this content in a user flow, an in-line error message, tool tip, field label?
     - This content is for updating [the FSR's 'Find a form' page](https://www.va.gov/find-forms/about-form-5655/) to provide more context about the form in alignment with other 'Find a form' pages and update the resources links to more relevant pages. 
   * **Scope: How many screens or messages?** 
     - 1            

- [ ] **Other**
   * I've synced with the Decision tools team so we're all looped in on this work. I'm happy to hop on a call to chat about the draft content!
      

### Desired/target implementation date

What's the ETA for the app/tool launch or the date when you need this request completed? Our team's capacity will determine which sprint the work falls into.

* The ETA for the FSR's launch is this sprint, but all debt is currently on hold due to COVID so we don't project an influx of FSR submissions until Fall 2021. We'd like to get this content up ASAP in accordance with with the public website team's schedule. 

**Note:** As a best practice, PW does not release new products over weekends or holidays. Please plan accordingly. 

---

### About your team

- Team name: Debt Resolution
- DEPO product owner: Matthew Self
- Product manager: Leah Keeler
- Designer: Riley Orr, Becca Walsh
- FE engineer: Craig Wheeler, Justin Linn
- Product/team Slack channel: vsa-debt

--- 
#### Acceptance Criteria
- [ ] Convert this request into a Zenhub epic
- [ ] Follow-up with requestor to address any lingering questions/concerns
- [ ] Create epic/tickets as needed for the Public Websites team to implement

#### Definition of Done
* Request ticket is closed and implementation tickets created as needed

**Comment by DanielleThierryUSDSVA** [comment]
Hi @rileyorr @denisecoveyduc @RLHecht. I believe this is blocked until all FSR bugs are resolved. So, just putting blocked label and moving this out of "in progress" as I'm going through some tickets. If anything's changed and it should move back into "in progress," let me know. Thanks!

**Comment by rileyorr** [comment]
@DanielleThierryUSDSVA I'm so glad this popped back up on our radar because I think we're planning to launch the FSR on Monday! @denisecoveyduc please correct me if I misunderstood the launch date - there's a lot of moving parts on our team right now

**Comment by denisecoveyduc** [comment]
@DanielleThierryUSDSVA Yes that is correct, late last week we conducted a final test and it appears that all issues have been resolved. There are some internal discussions about systems migrations that should have happened last week but our goal is to launch (hopefully today- Monday 11/15) but we are waiting to hear back from Jill. I will keep you posted, opening a slack thread on the PW channel now to also make @brianalloyd aware ;)

**Comment by rileyorr** [comment]
Hi @DanielleThierryUSDSVA! This work slipped my radar, but I believe this ticket is now unblocked since we've launched the FSR. Just as a refresher, this ticket is related to updating [the FSR's find a form page](https://staging.va.gov/find-forms/about-form-5655/) to incorporate content that Veterans can now fill out the FSR online. 

cc @denisecoveyduc

**Comment by RLHecht** [comment]
@SgardnerPMP Our team can work on this one. Tagging you here so you can set up the tickets we will need.

**Comment by RLHecht** [comment]
Work complete. Closing ticket.
